install_config/install/disconnected_install.adoc

```diff
@@ -0,0 +1,550 @@
+# docker tag registry.access.redhat.com/rhscl/php-56-rhel7 $REGISTRY:5000/openshift/php-56-rhel7
+# docker tag registry.access.redhat.com/rhscl/postgresql-94-rhel7 $REGISTRY:5000/openshift/postgresql-94-rhel7
+# docker tag registry.access.redhat.com/rhscl/python-27-rhel7 $REGISTRY:5000/openshift/python-27-rhel7
+# docker tag registry.access.redhat.com/rhscl/python-34-rhel7 $REGISTRY:5000/openshift/python-34-rhel7
+# docker tag registry.access.redhat.com/rhscl/ruby-22-rhel7 $REGISTRY:5000/openshift/ruby-22-rhel7
+# docker tag registry.access.redhat.com/openshift3/nodejs-010-rhel7 $REGISTRY:5000/openshift/nodejs-010-rhel7 
+----
+
+[[disconnected-create-an-admin-user]]
+=== Create an Administrative User
+
+Pushing the Docker images into OpenShift’s Docker registry requires a user with administrator privileges. Because the default OpenShift system administrator does not have a standard authorization token, they cannot be used to log in to the Docker registry.
+
+Create a new user account in the authentication system you are using with
+OpenShift. For example, if you are using local htpasswd-based authentication:
+
+----
+# htpasswd -b /etc/openshift/openshift-passwd <username> <password>
+----
+
+The external authentication system now has a user account, but a user must log in to OpenShift before an account is created in the internal database. Log in to OpenShift for this account to be created. This assumes you are using the self-signed certificates generated by OpenShift during the installation:
+
+----
+# oc login --certificate-authority=/etc/origin/master/ca.crt \
+    -u adminuser https://openshift.master.host:8443
+----
+
+Get the user’s authentication token:
+
+----
+# MYTOKEN=$(oc whoami -t)
+# echo $MYTOKEN
+iwo7hc4XilD2KOLL4V1O55ExH2VlPmLD-W2-JOd6Fko
+----
+
+[[disconnected-modify-the-securitry-policies]]
+=== Modify the Security Policies
+
+. Using `oc login` switches to the new user. Switch back to the OpenShift system administrator in order to make policy changes:
++
+----
+# oc login -u system:admin
+----
+
+. In order to push images into the OpenShift Docker registry, an account must
+have the image-builder security role. Add this to your OpenShift admin user:
++
+----
+# oadm policy add-role-to-user system:image-builder <admin_username>
+----
+
+. Next, add the administrative role to the user in the openshift project. This will allow the admin user to be able to edit the openshift project, and, in this case, push the Docker images:
++
+----
+# oadm policy add-role-to-user admin <admin_username> -n openshift
+----
+
+[[disconnected-edit-the-imagestream-definitions]]
+=== Edit the Imagestream Definitions
+
+The openshift project is where all of the imagestreams for builder images are created by the installer. They are loaded by the installer from the *_/usr/share/openshift/examples_* directory. You must change all of the definitions, by deleting which imagestreams had been loaded into OpenShift’s database, then re-create them.
```

Comment:
If "openshift" is a technical term (i.e., the name of the project), suggest backticks as markup.  Otherwise, suggest pretty name "OpenShift".